src/javascripts/views/homeView.js

```diff
@@ -51,7 +52,8 @@ class HomeView {
     }
 
     renderCategory(category, transactions, totalOutFlow) {
-       const categoryTemplate = Template.displayCategory(category, transactions, totalOutFlow);
+       const categoryTemplate = TransactionTemplate.displayCategory(category, transactions, totalOutFlow);
```

Comment:
IMO, display mean you will show it on UI. But in this case, you create template, not display

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks